platform/prologue/modfx/Makefile

```diff
@@ -28,9 +28,11 @@ ZIP_ARGS = -r -m -q
 
 ifeq ($(OS),Windows_NT)
 ifneq ($(MSYSTEM), MSYS)
+ifneq ($(MSYSTEM), MINGW64)
```

Comment:
I ran into this issue as well using Mingw 64bit.

I hacked around it by sym linking zip to $TOOLSDIR/zip/bin/zip, but your solution is better.